packages/accounts-base/accounts_client.js

```diff
@@ -235,11 +246,18 @@ makeClientLoggedIn = function(userId, token, tokenExpires) {
  * @param {Function} [callback] Optional callback. Called with no arguments on success, or with a single `Error` argument on failure.
  */
 Meteor.logout = function (callback) {
-  Accounts.connection.apply('logout', [], {wait: true}, function(error, result) {
+  return Accounts.logout(callback);
+};
```

Comment:
Here's a good example of a general principle I tried to follow: wherever there was a `Meteor.*` method that was implemented in terms of `Accounts`, I moved as much logic as possible into a method of `AccountsClient` or `AccountsServer`, and simplified the `Meteor.method` accordingly.

Changing the `Meteor.*` API is out of scope for this pull request, but at least we can reduce the difficulty of reimplementing what it does by making it a thinner wrapper around the `AccountsClient` and `AccountsServer` APIs.